tests/integration/components/gleam-logo-test.ts

```
@@ -0,0 +1,224 @@
+
+    // Verify canvas is in the DOM
+    assert.dom('canvas').exists('Canvas element is rendered');
+
+    // Get the container and create mock Rive instance
+    const container = document.querySelector('.gleam-logo-container') as HTMLElement;
+    const canvas = container?.querySelector('canvas');
+    assert.ok(canvas, 'Canvas element exists');
+
+    // Create and attach mock Rive instance
+    const mockRive = new MockRive({ canvas: canvas as HTMLCanvasElement });
+    (container as any).__riveInstance = mockRive;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)